### PR TITLE
f-vue-icons@0.13.0 tests for AppStoreIcon and BaseProviderIcon components

### PR DESCRIPTION
**Pull request by dandel10n**

### CODE REVIEW

**Review by DamianMullins — APPROVED** [review]

**Review by sonnyprince — APPROVED** [review]

File: packages/f-vue-icons/src/components/AppStoreIcon/AppStoreIcon.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { shallowMount } from '@vue/test-utils';
+import AppStoreIcon from './AppStoreIcon.vue';
+
+let propsData = { name: 'ios' };
```
Comment (kevinrodrigues) [inline packages/f-vue-icons/src/components/AppStoreIcon/AppStoreIcon.test.js:4]: This should really live in a `beforeEach` within the main `describe` below.

File: packages/f-vue-icons/src/components/AppStoreIcon/AppStoreIcon.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+describe('AppStoreIcon', () => {
+    it('should be defined', () => {
+        const wrapper = shallowMount(AppStoreIcon, {
```
Comment (kevinrodrigues) [inline packages/f-vue-icons/src/components/AppStoreIcon/AppStoreIcon.test.js:8]: Minor but missing the arrange act comment.

File: packages/f-vue-icons/src/components/AppStoreIcon/AppStoreIcon.test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            propsData
+        });
+        expect(wrapper).toBeDefined();
+    });
+
+    it('default component markup is rendered', () => {
```
Comment (kevinrodrigues) [inline packages/f-vue-icons/src/components/AppStoreIcon/AppStoreIcon.test.js:14]: Another minor point but these all should ideally start with the word `should`, just helps when reading it back. 

`AppStoreIcon  > should render the default markup component`

File: packages/f-vue-icons/src/components/BaseProviderIcon/BaseProviderIcon.test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { shallowMount } from '@vue/test-utils';
```
Comment (kevinrodrigues) [inline packages/f-vue-icons/src/components/BaseProviderIcon/BaseProviderIcon.test.js:1]: Similar comments to the other test file here.

**Pull request merged**